features/edit_company.feature

```diff
@@ -9,6 +9,10 @@ Feature: As a member
       | applicant_3@happymutts.com |       | false     |
       | admin@shf.se               | true  | true      |
 
+    Given the following payments exist
+      | user_email                 | start_date | expire_date | payment_type | status | hips_id |
+      | applicant_1@happymutts.com | 2017-10-1  | 2017-12-31  | member_fee   | betald | none    |
```

Comment:
Use a more meaningful email name so that the intent/role/purpose of the user is clear both in the scenarios and in looking at the raw data (e.g. in the db tables).

How about "paid_member@happymutts.com" ?  That will make the difference between this user & the other one obvious.  (one is paid, the other isn't)